packages/patterns/package.json

```diff
@@ -32,21 +28,25 @@
     "lint:types": "tsc -p jsconfig.json",
     "test": "ava"
   },
-  "dependencies": {},
+  "dependencies": {
+    "@endo/eventual-send": "^0.16.8",
+    "@endo/marshal": "^0.8.2",
+    "@endo/promise-kit": "^0.2.52"
+  },
   "devDependencies": {
-    "@endo/eslint-config": "^0.3.6",
+    "@endo/eslint-config": "^0.5.2",
     "@endo/init": "^0.5.53",
     "@endo/ses-ava": "^0.2.37",
     "ava": "^5.1.0",
     "babel-eslint": "^10.0.3",
-    "eslint": "^7.23.0",
+    "eslint": "^7.32.0",
     "eslint-config-airbnb-base": "^14.0.0",
     "eslint-config-prettier": "^6.9.0",
     "eslint-plugin-eslint-comments": "^3.1.2",
-    "eslint-plugin-import": "^2.19.1",
+    "eslint-plugin-import": "^2.26.0",
     "eslint-plugin-prettier": "^3.4.1",
     "prettier": "^2.8.0",
-    "typescript": "~4.8.4"
+    "typescript": "~4.9.4"
```

Comment:
@kriskowal 
Thanks for https://www.youtube.com/watch?v=SCOYsE5G7Xk. It is how I found out how to update the version numbers, and is what clued me in to put this tilda back in afterwards.